tests/adapter/dbt/tests/adapter/constraints/test_constraints.py

```diff
@@ -32,7 +34,35 @@ def models(self):
             "constraints_schema.yml": model_schema_yml,
         }
 
-    def test__constraints_wrong_column_order(self, project):
+    @pytest.fixture
+    def string_type(self):
+        return "TEXT"
+
+    @pytest.fixture
+    def int_type(self):
+        return "INT"
+
+    @pytest.fixture
+    def schema_int_type(self, int_type):
+        return int_type
+
+    @pytest.fixture
+    def data_types(self, schema_int_type, int_type, string_type):
```

Comment:
I would have loved to use pytest's `parametrized` functionality here (and did implement it that way to start), but unfortunately couldn't find a way to make it work with the testing inheritance pattern we have for adapter tests given the parametrized array would need to be provided dynamically - which pytest.parametrize does not support.